e2e/rbd_helper.go

```diff
@@ -279,6 +268,37 @@ func validateEncryptedPVCAndAppBinding(pvcPath, appPath, kms string, f *framewor
 	return nil
 }
 
+func validateEncryptedPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim, app *v1.Pod) error {
+	imageData, err := getImageInfoFromPVC(pvc.Namespace, pvc.Name, f)
+	if err != nil {
+		return err
+	}
+	rbdImageSpec := imageSpec(defaultRBDPool, imageData.imageName)
+
+	return validateEncryptedImage(f, rbdImageSpec, app)
+}
+
+func validateEncryptedImage(f *framework.Framework, rbdImageSpec string, app *v1.Pod) error {
```

Comment:
@nixpanic please add some source code comment about this function.

Reply:
will do!